src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Requests/DataSetQuerySort.cs

```diff
@@ -35,12 +35,12 @@ public record DataSetQuerySort
 
     public static DataSetQuerySort FromString(string sort)
     {
-        var parts = sort.Split('|');
+        var directionDelimiter = sort.LastIndexOf('|');
 
         return new DataSetQuerySort
         {
-            Field = parts[0],
-            Direction = parts[1]
+            Field = sort[..directionDelimiter],
+            Direction = sort[(directionDelimiter + 1)..]
```

Comment:
Was this change necessary or did you just think it was nicer? Doesn't look necessary to me but I don't mind either way

Reply:
Yep this is necessary as I've changed up sort fields for locations. This is now `location|{level}|{direction}` e.g. `location|LA|Desc`, `location|REG|Desc`, etc.

This means there can now be either 2 or 3 `|` characters in the sort string.